tensorflow/compiler/xla/service/user_computation.cc

```diff
@@ -421,7 +421,8 @@ StatusOr<ComputationDataHandle> UserComputation::AddMapInstruction(
       to_apply_computation.ComputeProgramShape(to_apply_version));
   TF_ASSIGN_OR_RETURN(
       Shape inferred_shape,
-      ShapeInference::InferMapShape(operand_shapes, *to_apply_program_shape));
+      ShapeInference::InferMapShape(operand_shapes, *to_apply_program_shape,
```

Comment:
@andydavis1 seeing a compilation error in https://ci.tensorflow.org/job/tensorflow-pull-requests-cpu/6991/consoleFull 

Any ideas on what the issue might be?